src/comm/S3.cpp

```diff
@@ -25,7 +25,7 @@ FMI::Comm::S3::S3(std::map<std::string, std::string> params, std::map<std::strin
     upload_price = std::stod(model_params["upload_price"]);
 
     auto credentialsProvider = Aws::MakeShared<Aws::Auth::EnvironmentAWSCredentialsProvider>(TAG);
```

Comment:
Do we still need this variable?

Reply:
No @mcopik. It should not be required because the latest S3 client has its own internal credentialsProvider